src/main/java/eu/elixir/ega/ebi/it/KeyBase.java

```diff
@@ -0,0 +1,32 @@
+package eu.elixir.ega.ebi.it;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.specification.RequestSpecification;
+
+public class KeyBase {
+    public RequestSpecification REQUEST;
+    String fileId;
+    String keyId;
+
+    public KeyBase() {
+        try {
+            Properties props = new Properties();
+            props.load(getClass().getClassLoader().getResourceAsStream("config.properties"));
```

Comment:
Instead of everyone having to modify the config.properties file to run the tests, you could create a maven profile https://www.mkyong.com/maven/maven-profiles-example/ with the default properties so people could override that properties from the command line

Reply:
Instead of creating maven profile; added the properties in pom.xml <systemPropertyVariables>